imports/ui/components/inactiveClusters/index.js

```diff
@@ -34,3 +35,22 @@ Template.inactiveClusters.onCreated(function() {
         this.subscribe('clusters.zombie', Session.get('currentOrgId'));
     });
 });
+
```

Comment:
after the prunes are complete can you also make a call to `updateResourceStats` meteor method?  That should make the cluster and resources numbers in the nav bar update without having to reload the browser

Reply:
2b8af2d48aedea09ea4ee20dfce37fe114119f79 added to do the requested action.